src/render_tree.rs

```diff
@@ -330,107 +349,4 @@ mod tests {
             panic!()
         }
     }
-
-    #[test]
-    fn text_nodes() {
```

Comment:
This test is moved to `gosub-bindings/tests`

Reply:
Not sure if you want to have this tests only over the C API

Reply:
yeah I was on the fence about this... my thought was the main purpose of this portion of the test was more relevant for the C API, and it might be good to have both tests but then I'd have to maintain two versions of the same test, and I'd rather have the C API be right (which would also imply the render tree implementation is correct.) But I can see the argument for having it in the render tree itself as well